standard/toolchain-android64.cmake

```diff
@@ -1,4 +1,3 @@
-set(ANDROID_NDK_TOOL_PREFIX aarch64-linux-android)
-set(_ndk_suffix -4.9-aarch64)
-set(CMAKE_SYSTEM_PROCESSOR aarch64)
+set(CMAKE_ANDROID_ARCH_ABI arm64-v8a)
 include(${CMAKE_CURRENT_LIST_DIR}/toolchain-android.cmake)
+
```

Comment:
Should have spotted this earlier, but now that standard is getting merged into other repos I'm noticing this stray extra newline.